pkg/dcmtime/dt.go

```diff
@@ -0,0 +1,77 @@
+package dcmtime
+
+import (
+	"fmt"
+	"time"
+)
+
+// DT holds data for a parsed DICOM datetime (DT value).
+type DT struct {
+	// Value as a native go time.Time value.
+	Time time.Time
+	// The precision with this value was stored. For instance, a DT value with a
+	// precision of Precision.Year ONLY stored the year.
+	Precision PrecisionLevel
+	// If true, offset information was specifically included in the DT string.
+	HasOffset bool
```

Comment:
as mentioned above, comments should start with the name of the identifer first.

For example

```
// Time ...
```

In the effective Go reccomendation:
> the first sentence should be a one-sentence summary that starts with the name being declared.

https://golang.org/doc/effective_go.html#commentary